.golangci.yml

```diff
@@ -0,0 +1,61 @@
+run:
+  tests: false
+  skip-dirs:
+  - "mocks"
+issues:
+  exclude-rules:
+    - path: internal/events/test_helper.go
+      text: "unused"
+linters-settings:
+  golint: {}
+  gocritic:
+    enabled-checks: []
+    disabled-checks:
+      - regexpMust
+  goheader:
+    values:
+      regexp:
+        COMPANY: .*
+    template: |-
+      Copyright © {{ YEAR }} {{ COMPANY }}
+      
+      SPDX-License-Identifier: Apache-2.0
+      
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+      
+          http://www.apache.org/licenses/LICENSE-2.0
+      
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+linters:
+  disable-all: false
+  disable:
+    - structcheck
+  enable:
+    # - depguard
```

Comment:
I had to disable this linter because it throws error for many packages being used. Maybe another issue is required to upgrade those packages which might require more fixes.

Reply:
I actually removed the `depguard` linter because I saw that it is also removed here: https://github.com/hyperledger/firefly/blob/main/.golangci.yml